bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -13,6 +13,21 @@
 			</add-channel>
 		</instruction-set>
 
+		<instruction-set targetVersion="2">
+			<add-channel id="grid-tariff" groupIds="electricity">
+				<type>energidataservice:datahub-price</type>
+				<label>Grid Tariff</label>
+				<description>Current grid tariff in DKK per kWh.</description>
+			</add-channel>
+			<add-channel id="transmission-grid-tariff" groupIds="electricity">
+				<type>energidataservice:datahub-price</type>
+				<label>Transmission Grid Tariff</label>
+				<description>Current transmission grid tariff in DKK per kWh.</description>
+			</add-channel>
```

Comment:
Why setting label/description ? Aren't they inherited from the channel type ?

Reply:
The channel type is generic for datahub price, so it's overloaded by the channels. See screenshots here: openhab/openhab-core#3660.

Reply:
Ok, same channel type. 
I hope I have not given wrong comment in other PRs when I asked to remove label/description

Reply:
It's a trade-off because you will lose translated texts when providing them explicitly in the update instructions. However, if you don't, texts will be taken from the channel type. So usually it's best to provide them when channel label/description is very different from channel type label/description.